airflow/lineage/datasets.py

```diff
@@ -77,6 +82,8 @@ def __iter__(self):
         yield from self._data.items()
 
     def as_dict(self):
+        """Return a dictionary containing two elements: typeName and attributes.
```

Comment:
When writing multi-line docs I think it is nicer to have the beginning and end quotes on separate lines. Don't you think?
```
"""
foo
bar
"""
```
instead of
```
"""foo
bar"""
```

Reply:
Thank you for the feedback, I agree, it's nicer and also better for readability